bin/node/src/chain_spec.rs

```diff
@@ -72,19 +73,19 @@ pub fn development_config() -> Result<ChainSpec, String> {
         .map(|bytes| AlephId::from(ed25519::Public::from_raw(bytes)))
         .collect();
 
-    let rich_accounts: Vec<_> = [
+    let mut rich_accounts: Vec<_> = [
         "Alice",
         "Alice//stash",
         "Bob",
         "Bob//stash",
         "Charlie",
         "Dave",
         "Eve",
-        "Ferdie",
     ]
     .iter()
     .map(get_account_id_from_seed::<sr25519::Public>)
     .collect();
+    rich_accounts.push(hex!["eaefd9d9b42915bda608154f17bb03e407cbf244318a0499912c2fb1cd879b74"].into());
```

Comment:
Would be nice to leave a comment or so, saying that "This is the faucet address", to not get confused later.

Reply:
Good idea, done.